src/gui/application.cpp

```diff
@@ -95,7 +95,8 @@ namespace {
         "  --localdirpath       : (optional) path where to create a local sync folder when creating an account via command-line.\n"
         "  --isvfsenabled       : whether to set a VFS or non-VFS folder (1 for 'yes' or 0 for 'no') when creating an account via command-line.\n"
         "  --remotedirpath      : (optional) path to a remote subfolder when creating an account via command-line.\n"
-        "  --serverurl          : a server URL to use when creating an account via command-line.\n";
+        "  --serverurl          : a server URL to use when creating an account via command-line.\n"
+        "  --forcelegacyimport  : forcefully import account configurations from legacy clients (if available).\n";
```

Comment:
@claucambra I would be more specific with argument names as it confuses me of what legacy import is this related to. Would be easier to read if the name was something like `--forcelegacyconfigimport`

Reply:
Okay, changed